versioned_docs/version-v6/api/toast.md

```diff
@@ -91,7 +91,7 @@ function Example() {
 
 ## Dismissing
 
-The toast can be dismissed automatically after a specific amount of time by passing the number of milliseconds to display it in the `duration` of the toast options. If a button with a role of `"cancel"` is added, then that button will dismiss the toast. To dismiss the toast after creation, call the `dismiss()` method on the instance.
+The toast can be dismissed automatically after a specific amount of time by passing the number of milliseconds to display it in the `duration` of the toast options. To dismiss the toast after creation, call the `dismiss()` method on the instance.
```

Comment:
Why was this removed? Buttons in toasts with a `cancel` role dismiss the toast: https://github.com/ionic-team/ionic-framework/blob/f68644065c227b73fe49330b030e2cfdad7b603a/core/src/components/toast/toast.tsx#L238-L240

Reply:
I misunderstood the wording and thought it was related to the hardware back button. I'll revert back.